packages/design-system/src/Button/Button.types.ts

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { Sizes } from "__config__/types";
 
 // Button types
-export type Kind = "primary" | "secondary" | "tertiary" | "error";
+export type ButtonKinds = "primary" | "secondary" | "tertiary" | "error";
```

Comment:
The plural of [X]Kind is generally [X]Kind - as in there's no change. Since we're primarily using these types to indicate that an acceptable value is "one of" these types, let's stick to the singular form?